.nvmrc

```diff
@@ -1 +1 @@
-v12.22.7
+14.18
```

Comment:
I think it’s recommended to pin to a patch version.

Also I prefer it with the `v` prefix 😄

Reply:
I started out with exactly that :)

I then noticed that DCR and the newer repos are only doing minor and dropping the v, so aligned with them.

Makes sense I think to get patch updates - if we trust them not make breaking changes.

Reply:
Someone (@sihil?) mentioned that breaking changes have previously found their way in Node patches… but 🤷